src/main/java/calculator/operator/OperatorEnum.java

```diff
@@ -0,0 +1,57 @@
+package calculator.operator;
+
+public enum OperatorEnum {
+
+    PLUS("+") {
+        public double calculate(double operand1, double operand2) {
+            return operand1 + operand2;
+        }
+    },
```

Comment:
```suggestion
    PLUS("+", (operand1, operand2) -> operand1 + operand2),
```
람다 표현식을 사용하여 보일러플레이트를 제거할 수 있습니다.